src/test/transaction_tests.cpp

```diff
@@ -560,8 +560,11 @@ BOOST_AUTO_TEST_CASE(test_IsStandard)
     string reason;
     BOOST_CHECK(IsStandardTx(t, reason));
 
-    t.vout[0].nValue = 53; // dust
-    BOOST_CHECK(!IsStandardTx(t, reason));
+    t.vout[0].nValue = 53; // dust in Bitcoin, but not in Zcash
+    BOOST_CHECK(IsStandardTx(t, reason));
+
+    t.vout[0].nValue = 0; // still not dust
```

Comment:
IMHO it's not useful to enforce minimum txout values, including that they are not zero, *as a consensus rule*. For efficiency reasons, higher-level APIs like ``sendmany`` and ``z_sendmany`` should not create them, but they still don't after this PR, unless I'm mistaken.